tests/test_box_utils.py

```diff
@@ -152,6 +156,22 @@ def test_value(self, input_data, mode2, expected_box, expected_area):
         expected_box_standard = convert_box_to_standard_mode(boxes=expected_box, mode=mode2)
         assert_allclose(result_standard, expected_box_standard, type_test=True, device_test=True, atol=0.0)
 
+        # test box_area, box_iou, box_giou, box_pair_giou
+        assert_allclose(box_area(result_standard), expected_area, type_test=True, device_test=True, atol=0.0)
```

Comment:
I was not sure which case you referred to. Could you help make it more specific? Thank you!

Reply:
yes, previously we discussed reusing `ensure_tuple` and there's an issue of not supporting this case:
```
>>> ensure_tuple(torch.zeros((3, )))
(tensor(0.), tensor(0.), tensor(0.))
```
(instead we only support `[0., 0., 0.]`)

I think the utils should be compatible with these different structure types.

Reply:
Yes. It is not used here in the current box_utils. I remember it might be used in transform or other funcs. I will keep it in mind when I see this func used. 
What do yo think?